ostelco-ios-client/ViewControllers/ESim/ESIMPendingDownloadViewController.swift

```diff
@@ -55,7 +55,7 @@ class ESIMPendingDownloadViewController: UIViewController {
             .onSuccess { data in
                 if let simProfiles: [SimProfile] = data.typedContent(ifNone: nil) {
                     if let simProfile = simProfiles.first(where: {
-                        $0.eSimActivationCode == self.simProfile?.eSimActivationCode
+                        $0.iccId == self.simProfile?.iccId
```

Comment:
might be worth throwing a `#warning` in here to fix this - this is definitely something we should fix before shipping